core/arch/arm/plat-synquacer/platform_config.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2018, Linaro Limited
+ * SPDX-License-Identifier: BSD-2-Clause
```

Comment:
Move to first line: `/* SPDX-License-Identifier: BSD-2-Clause */`

Reply:
Shall I add `// SPDX-License-Identifier: BSD-2-Clause` instead in first line?

Reply:
This is a header file so `/* SPDX-License-Identifier: BSD-2-Clause */` (C-style comment) in first line please

Reply:
ok